.gitignore

```diff
@@ -55,3 +55,4 @@ aesara-venv/
 testing-report.html
 coverage.xml
 .coverage.*
+jax_multinomial_test.py
```

Comment:
This looks like a job for a "local" Git ignore (see [here](https://stackoverflow.com/a/1753078/3006474)).